client/state/ui/editor/media/imageEditor/constants.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import keyMirror from 'key-mirror';
+
+export const AspectRatios = keyMirror( {
+	FREE: null,
+	ORIGINAL: null,
+	ASPECT_1x1: null,
```

Comment:
Consistency on the casing is off here, `X` vs. `x` below (should probably be capitalized for all)